layout/page.ftd

```diff
@@ -37,6 +37,7 @@ ftd.length.px spacing: 0
 page-widths widths: $widths
 ;; todo: is the follow reference to `.gap` correct?
 ds.gap margin: $ds.spaces.vertical-gap.large
+optional ftd.background.solid background: $ds.colors.background.base
```

Comment:
If you are passing a default value you do not have to call it `optional`. Use `optional` only when you are not specifying a default value.